core/templates/domain/learner_dashboard/learner-playlist-modal.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="modal-header">
+    <h3 [innerHTML]="'I18N_LEARNER_DASHBOARD_REMOVE_ACTIVITY_MODAL_HEADER' | translate" translate-values="{sectionNameI18nId: sectionNameI18nId}"></h3>
+  </div>
+  
+  <div class="modal-body">
+    <p [innerHTML]="'I18N_LEARNER_DASHBOARD_REMOVE_ACTIVITY_MODAL_BODY' | translate" translate-values="{entityTitle: activityTitle, sectionNameI18nId: sectionNameI18nId}"></p>
+  </div>
+  
+  <div class="modal-footer">
+    <button class="btn btn-secondary" (click)="cancel()" translate="I18N_MODAL_CANCEL_BUTTON"></button>
```

Comment:
Should this translate tag also be changed as above?

Reply:
resolved